src/Migrator/TemporalClientExtensions.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) InfinityFlow. All Rights Reserved.
+// Licensed under the Apache 2.0. See LICENSE file in the solution root for full license information.
+
+namespace Temporalio.Client;
+
+using System.Reflection;
+using InfinityFlow.Temporal.Migrator;
+
+/// <summary>
+/// Temporal Client Extensions.
+/// </summary>
+public static class TemporalClientExtensions
+{
+    /// <summary>
+    /// Bootstrap Migrations.
+    /// </summary>
+    /// <param name="temporalClient">The temporal client.</param>
+    /// <param name="workflowOptions">The workflow options.</param>
+    /// <param name="assemblies">The assemblies.</param>
+    /// <returns>The <see cref="Task"/>.</returns>
+    public static Task RunMigrator(this ITemporalClient temporalClient, WorkflowOptions workflowOptions, params Assembly[] assemblies)
+    {
+        ArgumentNullException.ThrowIfNull(temporalClient);
+        var assemblyParams = assemblies.Select(s => s.ToString());
+        return temporalClient
```

Comment:
We should probably return a workflow handle here so this doesn't become a blocking call and the caller can decide if they want to block or not. See StartChildWorkflow